docs/learning/frontend/Localization.md

```diff
@@ -4,23 +4,25 @@ Presenting information to the user in their preferred locale (language, date and
 
 ## Language translation
 
-String localization is handled in a conventional way. Rather than specifying strings directly, a "key" is passed to the `IModelApp.localization.getLocalizedString` method, which retrieves the corresponding string for the current locale for presentation to the user.
+String localization is handled in a conventional way. Rather than specifying strings directly, a "key" is passed to the `Localization.getLocalizedString` method, which retrieves the corresponding string for the current locale for presentation to the user.
```

Comment:
Use doc links throughout please. e.g., in this case [Localization.getLocalizedString]($common).